<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QSlider" name="horizontalSlider_position">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>411</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_volume">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>110</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>播放进度：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>音量控制：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_play">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pause">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_stop">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>媒体状况</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_state">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>播放状态</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_error">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>60</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>错误信息</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
